client/my-sites/customer-home/cards/tasks/site-setup-checklist-ecommerce/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'my-sites/customer-home/cards/tasks/task';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteUrl } from 'state/sites/selectors';
+import { TASK_SITE_SETUP_CHECKLIST_ECOMMERCE } from 'my-sites/customer-home/cards/constants';
+
+export const SiteSetupListEcommerce = ( { siteUrl } ) => {
+	const translate = useTranslate();
+
+	return (
+		<Task
```

Comment:
Do you want to add the `timing` prop, to give users an idea of how long the task would take?

<img width="600" alt="Screen Shot 2020-09-08 at 11 42 30 AM" src="https://user-images.githubusercontent.com/349751/92515443-795b4800-f1c8-11ea-9ed2-4d9fd2c3c706.png">

Reply:
Thank you for the suggestion @kwight, I like the idea but I'm not sure about how much time to show there, since the task list has a few tasks and we're not able to know how many the user needs to finish. If it's only one, they could finish it in a minute or two but if they have all of them undone for sure it will take more time.

Reply:
We don't aim for accuracy – it's more like "hey, it'll probably take you about 10 minutes if you do it all right now". But certainly up to you if you thinking dropping it is the way to go.

Reply:
@pmcpinto what do you think about the estimated time for the `Finish store setup` card?
Is it ok if we add a fixed 7 minutes to this task?

Reply:
@octaedro 7 min sounds good, as it's the sum of the estimated time of the main tasks (products, shipping, taxes, payments, customize store)

Reply:
Added the timer prop with 7 minutes in the commit b8235d4